sdks/python/apache_beam/pipeline_test.py

```diff
@@ -61,39 +60,9 @@
 from apache_beam.utils import windowed_value
 from apache_beam.utils.timestamp import MIN_TIMESTAMP
 
-# TODO(BEAM-1555): Test is failing on the service, with FakeSource.
 
-
-class FakeSource(NativeSource):
-  """Fake source returning a fixed list of values."""
-  class _Reader(object):
-    def __init__(self, vals):
-      self._vals = vals
-      self._output_counter = Metrics.counter('main', 'outputs')
-
-    def __enter__(self):
-      return self
-
-    def __exit__(self, exception_type, exception_value, traceback):
-      pass
-
-    def __iter__(self):
-      for v in self._vals:
-        self._output_counter.inc()
-        yield v
-
-  def __init__(self, vals):
-    self._vals = vals
-
-  def reader(self):
-    return FakeSource._Reader(self._vals)
-
-
-class FakeUnboundedSource(NativeSource):
+class FakeUnboundedSource(SourceBase):
```

Comment:
Do we have any integration test to cover these changes? Since we are removing `native_io`, shall we update `CHANGES.md` for 2.50.0?

Reply:
native_io is an internal implementation detail of Runner v1.

Reply:
Probably good to update CHANGES.md since we are deleting some public APIs here (though we don't expect anyone other than Dataflow to have used these).

Reply:
This is not a public API. (It was explicitly marked "internal only" and sits inside the runners/dataflow directory.) It also doesn't work on any other distributed runners or Dataflow Runner v2. I decided to leave it out of CHANGES.md as I don't think it'd increase the signal to noise ratio there.